python-package/lightgbm/engine.py

```diff
@@ -408,6 +415,8 @@ def cv(params, train_set, num_boost_round=100,
     callbacks : list of callables or None, optional (default=None)
         List of callback functions that are applied at each iteration.
         See Callbacks in Python API for more information.
+    disp_train_loss : bool, optional　(default=False)
```

Comment:
disp_train_loss --> show_train_loss

Reply:
I agree to https://github.com/Microsoft/LightGBM/pull/2089#issuecomment-481918964.
I think`eval_train_loss` is better than `show_train_loss`, because the prameter will actually evaluate (calculate) train loss, not show (display) it.

Reply:
@henry0312 I think it should be `metric`, not `loss`. WDYT?

Reply:
Changed the aug name show_train_loss -> eval_train_metric. I also think 'metric' is match in this case.